src/components/Paper.jsx

```diff
@@ -43,20 +45,47 @@ export function Papers() {
                 de apresentar o teu paper aos participantes da JORDI!
               </p>
             </div>
-            <form className="ml-36">
-              <h3 className="text-lg font-semibold tracking-tight text-blue-900">
-                Submete o Paper aqui{' '}
-                <span className="hidden sm:inline-flex" aria-hidden="true">
-                  &darr;
-                </span>
-              </h3>
-              <Button type="submit" className="ml-20 mt-2 sm:ml-24" href="/">
-                <span className="sr-only sm:not-sr-only">Submissão</span>
-                <span className="sm:hidden">
-                  <ArrowRightIcon className="h-6 w-6" />
-                </span>
-              </Button>
-            </form>
+            <div className="flex flex-col items-center gap-10">
+              <form>
+                <h3 className="inline-block text-lg font-semibold tracking-tight text-secondary sm:block">
+                  Submete o Paper aqui{' '}
+                  <span className="hidden sm:inline-flex" aria-hidden="true">
+                    &darr;
+                  </span>
+                </h3>
+                <Button type="submit" className="ml-5 mt-2 sm:ml-24" href="/">
+                  <span className="sr-only sm:not-sr-only">Submissão</span>
+                  <span className="sm:hidden">
+                    <ArrowRightIcon className="h-6 w-6" />
+                  </span>
+                </Button>
+              </form>
+              <div className="mt-16 flex flex-col">
+                <h1 className="text-base font-semibold tracking-tighter text-secondary">
+                  Mais informações
```

Comment:
```suggestion
                  Mais informações{' '}
```

In order to have a space between the text and the arrow 🙏🏼